server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/VacationResponseGetMethodContract.scala

```diff
@@ -31,10 +31,23 @@ import org.apache.james.jmap.api.vacation.{AccountId, VacationPatch}
 import org.apache.james.jmap.draft.JmapGuiceProbe
 import org.apache.james.jmap.http.UserCredential
 import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.jmap.rfc8621.contract.VacationResponseGetMethodContract.VACATION_RESPONSE
 import org.apache.james.jmap.rfc8621.contract.tags.CategoryTags
 import org.apache.james.utils.DataProbeImpl
 import org.junit.jupiter.api.{BeforeEach, Tag, Test}
 
+object VacationResponseGetMethodContract {
+  private val VACATION_RESPONSE: VacationPatch =
+    VacationPatch.builder
+      .isEnabled(true)
+      .fromDate(ZonedDateTime.parse("2014-09-30T14:10:00+02:00"))
+      .toDate(ZonedDateTime.parse("2016-04-15T11:56:32.224+07:00[Asia/Vientiane]"))
```

Comment:
Just wonder why `Asia/Vientiane`

Reply:
Because it is hanoi timezone ;-)

Reply:
Why not?

Reply:
Just to show that you can put any time with any timezone, the UTC equivalent one will always be the one returned through the Serializer class :)